action/protocol/account/util/util.go

```diff
@@ -86,18 +86,24 @@ func Recorded(sr protocol.StateReader, addr address.Address) (bool, error) {
 	return false, err
 }
 
-// AccountState returns the confirmed account state on the chain
-func AccountState(sr protocol.StateReader, encodedAddr string) (*state.Account, error) {
-	a, _, err := AccountStateWithHeight(sr, encodedAddr)
+// AccountState returns the confirmed account state on the chain by address.Address
+func AccountState(sr protocol.StateReader, addr address.Address) (*state.Account, error) {
+	a, _, err := AccountStateWithHeight(sr, addr)
 	return a, err
 }
 
-// AccountStateWithHeight returns the confirmed account state on the chain with what height the state is read from.
-func AccountStateWithHeight(sr protocol.StateReader, encodedAddr string) (*state.Account, uint64, error) {
+// AccountStateByHash160 returns the confirmed account state on the chain by 20-byte address
```

Comment:
delete this func, we change the input from `string` to `address.Address`

Reply:
done